examples/model_assisted_labeling/tiled_imagery_mal.ipynb

```diff
@@ -35,34 +35,34 @@
   },
```

Comment:
<p>I don't think that these are all required. They should just be part of labelbox data. Which btw you have to install since we are using annotation types. Also we are now adding the -q flag so the notebook doesn't have tons of extra logs in it.</p><p> </p><p><code>pip install -q labelbox[data] </code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>